rfc000-rfc-process.md

```diff
@@ -124,6 +136,7 @@ RFC: unassigned
 Author: Alan Smithee <asmithee@example.com>
 Status: Draft
 Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
```

Comment:
We document "replaced" elsewhere in RFC-000, for consistency IMO it makes sense to include it here too